### DISCUSSION TITLE
How can I filter my data from table like `select * from table_name where column_name between a and b`

### DISCUSSION THREAD

**Discussion by Alkaidcc** [supabase/supabase#3808] [Questions] [answered]
The filter example I can see in bash is:
```
WITH FILTERING
curl 'https://fccmwidxdzbiuypgjzaz.supabase.co/rest/v1/Ticket?id=eq.1&select=*' \
-H "apikey: SUPABASE_KEY" \
-H "Authorization: Bearer SUPABASE_KEY" \
-H "Range: 0-9"
```
Can anyone tell me how to filter data like `select * from table_name where column_name between a and b` in bash way?
And I can't find `between` or other similar term in docs.
![image](https://user-images.githubusercontent.com/54631354/140601757-55ca29ec-2499-41fb-98a8-46a36a7c7115.png)
I have tried to pass params like below in Postman.But it seems dosen't work:
![image](https://user-images.githubusercontent.com/54631354/140602041-9433c858-1d82-46d4-8ace-42ea04875ead.png)

**Comment by GaryAustin1** [comment, marked as answer]
Not super clear on what you are asking but I use:
.gte('rating',3).lte('rating',6) in the JS and it goes out as &rating=gte.3&rating=lte.6 in the URL
if that helps.
Although you are showing example with time involved so the value format is going to be different for sure

**Reply by Alkaidcc** [reply]
I found solution at https://github.com/supabase/supabase/discussions/860#discussioncomment-960657 and https://github.com/supabase/supabase/discussions/1069#discussioncomment-574435
`between` can be implemented with expression like `and=(start_time.gt.06:00,start_time.lt.08:00)`
And I tried it in Postman, it does work.
![image](https://user-images.githubusercontent.com/54631354/140628995-3b841296-5043-4e89-a8f3-e81c73463e22.png)

Really thanks for your generous help！！
